qemu/tests/numa_memdev_options.py

```diff
@@ -111,10 +124,29 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment
     """
-    error_context.context("Check host's numa node(s) amount!", logging.info)
-    check_host_numa_node_amount(test)
-
+    error_context.context("Check host's numa node(s)!", logging.info)
+    node_list_size_not_zero = check_host_numa_node(test)
```

Comment:
Maybe we can use "valid_nodes"?

Reply:
How about we move the if block in L31 out  of the function and put it here?

Reply:
Yes, already update